hazelcast/src/main/java/com/hazelcast/spi/properties/ClusterProperty.java

```diff
@@ -1029,8 +1029,6 @@ private int getWhenNoSSLDetected() {
     public static final HazelcastProperty CLIENT_PROTOCOL_UNVERIFIED_MESSAGE_BYTES =
             new HazelcastProperty("hazelcast.client.protocol.max.message.bytes", 4096);
 
-    public static final HazelcastProperty AUDIT_LOG_ENABLED = new HazelcastProperty("hazelcast.auditlog.enabled", false);
```

Comment:
What if any users were already using this property to set it to true?

Reply:
It's not a documented property. It was intended only for API testing. 
I would rather see it removed, but if you think it can cause troubles we can leave it unused and just mark it deprecated.

Reply:
If it wasn't documented in the reference manual and if it was intended for testing, I think we can remove it.